config/jobs/kubernetes/sig-windows/windows-gce.yaml

```diff
@@ -19,6 +19,24 @@ presets:
     value: "containerd"
   - name: PREPULL_TIMEOUT
     value: "30m"
+- labels:
+    preset-load-gce-windows: "true"
+  env:
+  - name: MASTER_SIZE
+    value: "n1-standard-4"
+  - name: NUM_WINDOWS_NODES
+    value: "1"
+  - name: NODE_SIZE
+    value: "n1-standard-8"
+  - name: NODE_DISK_SIZE
+    value: "100GB"
+  # Ensure good enough architecture for master machines.
+  - name: MASTER_MIN_CPU_ARCHITECTURE
+    value: "Intel Broadwell"
+  # Bump max pods per node in Kubelet, because there are more than 10
+  # system pods in 1-node cluster.
+  - name: MAX_PODS_PER_NODE
+    value: "128"
```

Comment:
referred [this](https://github.com/kubernetes/test-infra/blob/9f320127fac5933d68849240028ef80a4dbf9f54/config/jobs/kubernetes/sig-scalability/sig-scalability-presets.yaml#L201), don't know the default value. Thought it might be needed, so add it here.

Reply:
ok

Reply:
I'm guessing this feeds into --max-pods for kubelet, which defaults to 110 (https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet/, https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/kubelet/config/v1beta1/types.go#L462).

Anyway, this value is fine but can you remove the comment? Maybe it made sense at some point but it does not make sense now.